### ISSUE TITLE
TTS-F: content1

### ISSUE THREAD

**Issue by hakolj** [l-O-O-l/TTS-action#67]
it is involuntary to use languages and follow certain rules that have been provided. 

However, language can convey a lot of information. 

Gendered language is a perpetuation of the historical patriarchal hierarchy. language can reflect a preconceived consciousness about social structure and even convey discrimination, which in return affecting the development of people's consciousness.

Under these circumstances, following the enactment of The Convention on the Elimination of All Forms of Discrimination against Women, the UN began to draft documents and revise each of its written statements and human rights documents to address masculine expressions that had been issued.

language policies of the United Nations are based on the concept of “language of human rights”. Article 1 of the UN chapter in 1945 clearly stated that "respecting and encouraging human rights and fundamental freedoms for all without distinction as to race, sex, language or religion". This shows that gender and language are regarded as the basic human rights. 

In addition, the emphasis of the Commission on the Status of Women that "women's rights are human rights" has further consolidated women's human rights in the context of universal human rights. 

And the concept of "gender mainstreaming" released on Third World Conference on Women in 1985 states that all policies and laws should reflect gender consciousness, has become the cornerstone of gender language policy. 

As a strategic approach, it makes the concerns and experiences of both males and females an integral part of the design, implementation, monitoring and evaluation of all policy programs in the political, economic and social fields in order to make sure men and women benefit equally. 

This is the framework of UN language policies.

As for the development of Guidelines, there are mainly three points

Firstly, the united states provides solutions to the pronouns.
Pronouns in language have been found to influence gender perception. Compared with masculine pronouns, data shows that neutral pronouns reduce individual psychological biases against men and enhance the salience of women in the minds of speakers. 

Therefore, UN did revision. For example, “The client should receive his invoice in two weeks” can be changed to “The client should receive his or her invoice in two weeks”. But it is a bit tedious and repetitive. Another good method is to reduce the use of personal pronouns by inversion. Such as “The invoice will be sent to the client in two weeks.” Lastly, it is feasible to replace with “the” or “an”. For example, “The client should receive the invoice in two weeks.” Moreover, according to the guidelines, the best solution is to change pronouns to plural. The United Nations also create “ze”, a neutral third person to show respects to all genders. 

Secondly, avoid highlighting gender-specific identities or roles.
Nouns are generally refer to the characteristics, such as a person's appearance, temperament, habits, experience, native place. These distinctive features i can inspire people's imagination in many aspects, which makes nouns have subjective and colorful connotations on the basis of objective and rational meanings.

Therefore, UN recommend using generic or gender-neutral terms, and try to avoid titles that exclude women or highlight masculinity. For example, "mankind" should be “humankind”, “humanity,” or “human beings”. And UN avoid “chairman”, but use “chairperson” or “chair” instead. UN uses “artificial” or “manufactured” instead of “man-made”.  Besides, UN has a complete term manual to revise every word and clearly indicate how to replace "Biased Word" with "Neutral Word". "comment" is used to explain the principle. 

Thirdly, UN highlight the independent identity of women. Language plays a particularly important role in shaping individuals' attitudes towards gender and occupation. Data also shows that when teachers used male terms to describe occupations, female schoolchildren perceived women as less successful in stereotypically male occupations, such as engineer, astronaut. women were significantly less likely to apply for jobs with male suffixes, such as fireman, Cameraman.
 
Under these circumstances, UN change “cameraman” to “photographer”, “fireman” to “firefighter”. Moreover, UN changed the sentence like “The Convention participants and their wives are invited” are changed to “The Convention participants and their spouses or guests are invited.” Another important rule to avoid showing a woman's marital status is to use “Ms.” to address a woman, rather than “Miss” or “Mrs.” 

In conclusion, with more than 50 years of practice, the United Nations has developed a unique and operational framework for action, including setting common standards and rules, promoting universal agreement, setting benchmarks and building evaluation mechanisms.
